Python/bltinmodule.c

```diff
@@ -326,8 +326,8 @@ should be a list of names to emulate ``from name import ...'', or an\n\
 empty list to emulate ``import name''.\n\
 When importing a module from a package, note that __import__('A.B', ...)\n\
 returns package A when fromlist is empty, but its submodule B when\n\
-fromlist is not empty.  Level is used to determine whether to perform \n\
-absolute or relative imports. 0 is absolute while a positive number\n\
+fromlist is not empty.  The level argument is used to determine whether to\n\
+perform absolute or relative imports, 0 is absolute while a positive number\n\
```

Comment:
With this change, I believe your new comma should be a semi-colon.  `imports; 0 is absolute`

Reply:
Isn't colon better than semi colon here?  BTW, should I add a new commit on top of this or amend this commit?

Reply:
We prefer you don't amend commits, so add a new one. Thanks.